### DISCUSSION TITLE
It suddently break my stats some weeks ago

### DISCUSSION THREAD

**Discussion by Add3rall** [anuraghazra/github-readme-stats#2956] [Q&A] [answered]
[![Add3rall's github stats](https://github-readme-stats.vercel.app/api?username=Add3rall&count_private=true&theme=tokyonight&hide=stars,prs,issues)](https://github.com/Add3rall/github-readme-stats

https://github.com/Add3rall/Add3rall

https://i.imgur.com/JUS8IT9.png

Somehow is not counting my commits, any clue?
I didnt report before because I though was temportal bug, but it persists.

**Comment by rickstaa** [comment, marked as answer]
@Add3rall Thanks for your question. It looks like the card is working correctly. You can check this yourself in the [GraphQL explorer](https://docs.github.com/en/graphql/overview/explorer).

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=Add3rall)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=Add3rall)](https://github.com/anuraghazra/github-readme-stats)
```

![image](https://github.com/anuraghazra/github-readme-stats/assets/17570430/a919aec8-56a4-4c7d-9390-78270b48a336)

We removed the `count_private` option since it counted Contributions instead of commits (see https://github.com/anuraghazra/github-readme-stats/pull/2736, https://github.com/anuraghazra/github-readme-stats/discussions/1679 and https://github.com/anuraghazra/github-readme-stats/issues/2517). If you want your private commits to appear, please [deploy your own Vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own).
